bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/photon_proton_NLO_MG_pp_13TeV/TTbar_photon_proton_NLO_MG_pp_13TeV/photo_NLO_M_ttbar_pp_13TeV.txt

```diff
@@ -0,0 +1,10 @@
+#import model sm-no_b_mass
```

Comment:
which model do you use for ttbar?

Reply:
In principle we used the sm default one, but we can use the sm-no_b_mass as for tW. tW is forced to that one otherwise it does not work at NLO or with madstr plugin